example.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+
+import boutvecma
+import easyvvuq as uq
+import chaospy
+import os
+import numpy as np
+import time
+
+campaign = uq.Campaign(name="Conduction")
+encoder = boutvecma.BOUTEncoder(template_input="models/conduction/data/BOUT.inp")
+decoder = boutvecma.BOUTDecoder(variables=["T"])
+params = {
+    "conduction:chi": {"type": "float", "min": 0.0, "max": 1e3, "default": 1.0},
+    "T:scale": {"type": "float", "min": 0.0, "max": 1e3, "default": 1.0},
+    "T:gauss_width": {"type": "float", "min": 0.0, "max": 1e3, "default": 0.2},
+    "T:gauss_centre": {"type": "float", "min": 0.0, "max": 2 * np.pi, "default": np.pi},
+}
+
+campaign.add_app("1D_conduction", params=params, encoder=encoder, decoder=decoder)
+
+vary = {
+    "conduction:chi": chaospy.LogUniform(np.log(1e-2), np.log(1e2)),
```

Comment:
Is it worth cutting the parameter space down so that we get something a little faster for a first example?

Reply:
Yes, probably have the serial version just do something simple that runs in a few minutes, tops. And then something a bit heavier for the parallel version?